ironfish/src/mining/stratum/stratumServer.ts

```diff
@@ -23,28 +23,30 @@ import {
   StratumMessage,
   StratumMessageSchema,
 } from './messages'
+import { StratumPeers } from './stratumPeers'
 import { StratumServerClient } from './stratumServerClient'
-import { STRATUM_VERSION_PROTOCOL_MIN } from './version'
+import { STRATUM_VERSION_PROTOCOL, STRATUM_VERSION_PROTOCOL_MIN } from './version'
+
+const FIFTEEN_MINUTES = 15 * 60 * 1000
```

Comment:
Very nit, so up to you, but I dig the `_MS` suffix for clarity on the consts in the other file.

Reply:
Fixed